misaligned_fixed&Test.py

```diff
@@ -0,0 +1,31 @@
+
+def print_color_map():
+    major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
+    minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
+    list = []
+
+    for i, major in enumerate(major_colors):
+        for j, minor in enumerate(minor_colors, 1):
+            interval = i * 5 + j
+            list.append(interval)
+            #assert list[0] == 0
+            #print(f'{interval} | {major} | {minor}')
+            #print("{:<7}|".format(interval), "{:<7}|".format(major), "{:<7}|".format(minor) )
+            #text = f'{interval} | {major} | {minor}'
+            #print(format(text, '=^20'))
+            space = 6
+            assert space >= 6, "Format not aligned"
+            #text = '| {:^6} | {:^6} | {:^6} |'.format(interval, major, minor)
+            #text = (f'{interval} | {major} | {minor}')
+            #print(text)
+            print('|',f'{interval}'.center(space),'|',f'{major}'.center(space),'|',f'{minor}'.center(space),'|')

```

Comment:
Nice use of python functionality to align :)

Reply:
haha Thanks! I learn about this writing alignment because of this task.... 
(Sorry for the commented lines I completely forgot them.... hehe)